### ISSUE TITLE
AI Code Review

### ISSUE THREAD

**Issue by blozano-tt** [tenstorrent/tt-metal#13623]
We find ourselves living in the era of large language models.

Should we integrate an AI code review bot, that could give feedback on code diffs?

Reference:

https://github.com/marketplace/actions/code-review-with-chatgpt

**Comment by bkeith-TT** [comment]
I agree with the prioritization of P3.  I support spending a timeboxed amount of time on a PoC.  Like with using CoPilot to generate code for automated tests, it's important to use this as an assistive device/tool and keep a human in the loop.

**Comment by blozano-tt** [comment]
We will never get to this, and clang-tidy comes first.
